tests/plain.js

```diff
@@ -35,5 +35,13 @@ QUnit.module('Тестируем функцию plain', function () {
 
 	QUnit.test('Работает с элементами разных типов', function (assert) {
 		assert.deepEqual(plain([ [ 'abcde' ], [ [ 'f' ], [ null, false ], [ NaN, NaN ], NaN ], -Infinity ]), [ 'abcde', 'f', null, false, NaN, NaN, NaN, -Infinity ]);
+		assert.deepEqual(plain([ undefined, [ null, "[[['abcd'],55]]", [ [ [3.14] ], [ false, true, [ 1488 ] ] ] ] ]), [undefined, null, "[[['abcd'],55]]", 3.14, false, true, 1488]);
+	});
+
+	QUnit.test('Работает с объектами', function (assert) {
```

Comment:
```suggestion
	QUnit.test('Работает с объектами', (assert) => {
```

Стрелочные функции forever!